tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -301,62 +318,77 @@ def notEmpty(start=0, end=None):
     """Validate that string value isn't only blanks."""
     return make_validator(
         lambda value: not _is_empty(value, start, end),
-        lambda value: f'{str(value)} contains blanks between positions {start} and {end if end else len(str(value))}.'
+        lambda value, record_type,
+        friendly_name, item_num: f'{record_type}: {str(value)} contains blanks between positions {start} and '
+                                 f'{end if end else len(str(value))}.'
+    )
+
+
+def caseNumberNotEmpty(start=0, end=None):
+    """Validate that string value isn't only blanks."""
+    return make_validator(
+        lambda value: not _is_empty(value, start, end),
+        lambda value, record_type,
+        friendly_name, item_num: f'{record_type}: Case number {str(value)} cannot contain blanks.'
```

Comment:
@reitermb can you verify this is appropriate language for the case number being incorrectly formatted?

Reply:
This looks good to me!